Parse/src/test/java/com/parse/ParseUserTest.java

```diff
@@ -165,12 +165,12 @@ public void testSignUpAsyncWithObjectIdSetAndAuthDataSet() throws Exception {
     ParseUser partialMockUser = spy(user);
     doReturn(Task.<Void>forResult(null))
         .when(partialMockUser)
-        .saveAsync(anyString(), any(Task.class));
+        .saveAsync(anyString(), Matchers.<Task<Void>>any());
```

Comment:
Incorrect generic since `saveAsync` requires `Task<Void>` but `any(Task.class)` is equivalent to `Task`.